src/tools/tests/recipe2plan-test.ts

```diff
@@ -7,16 +7,36 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
+import protobuf from 'protobufjs';
 import {assert} from '../../platform/chai-web.js';
 import {fs} from '../../platform/fs-web.js';
-import {recipe2plan} from '../recipe2plan.js';
+import {recipe2plan, OutputFormat} from '../recipe2plan.js';
 import {Flags} from '../../runtime/flags.js';
 
+const rootNamespace = protobuf.loadSync('./java/arcs/core/data/proto/manifest.proto');
+const manifestProto = rootNamespace.lookupType('arcs.ManifestProto');
```

Comment:
I left a comment in your CL about making a utility function to replace protobuf.loadSync. Now that I'm looking at this code again, it seems that helper code to wrap this all could be quite useful indeed. We could centralise the proto-loading in one place, with caching and lazy loading, and use some proper types for this instead of just strings. 

e.g. this would be a nice usage:

```
import {ManifestProto} from 'arcs-proto.ts';

const proto = ManifestProto.decode(encodedStr);
```

And then invoking that type would lazily load the proto file and return the correct type.

Doesn't need to be in this PR, but food for thought.

Reply:
Good stuff! Done.

Reply:
Pulled into https://github.com/PolymerLabs/arcs/pull/5247